lib/components/user/stacked-panes-with-save.tsx

```diff
@@ -78,4 +65,4 @@ const StackedPaneDisplay = ({
     </>
   )
 }
-export default StackedPaneDisplay
+export default StackedPanesWithSave
```

Comment:
Why do we do this instead of adding a prop? Is it formik nonsense?

Reply:
That's to separate the code noise brought by handling the navigation button states.